src/components/Navbar/styles.module.css

```
@@ -0,0 +1,9 @@
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: 1;
+    box-shadow: 0px 4px 4px 0px rgba(227, 227, 227, 0.25);
```

```suggestion
    box-shadow: 0px 4px 4px 0px rgba(227, 227, 227, 0.25);
```
Why there is no token for box shadow?

---

still need to finalize the token with the design team